spaceship/spec/spec_helper.rb

```diff
@@ -63,6 +64,21 @@ def before_each_spaceship
   TunesStubbing.itc_stub_generate_promocodes
   TunesStubbing.itc_stub_promocodes_history
   TunesStubbing.itc_stub_supported_countries
+
+  ConnectAPIStubbing.stub_apps
+  ConnectAPIStubbing.stub_beta_app_localizations
+  ConnectAPIStubbing.stub_beta_app_review_details
+  ConnectAPIStubbing.stub_beta_app_review_submissions
+  ConnectAPIStubbing.stub_beta_build_localizations
+  ConnectAPIStubbing.stub_beta_build_metrics
+  ConnectAPIStubbing.stub_beta_groups
+  ConnectAPIStubbing.stub_beta_testers
+  ConnectAPIStubbing.stub_beta_tester_metrics
+  ConnectAPIStubbing.stub_build_beta_details
+  ConnectAPIStubbing.stub_build_deliveries
+  ConnectAPIStubbing.stub_builds
+  ConnectAPIStubbing.stub_pre_release_versions
+  ConnectAPIStubbing.stub_users
```

Comment:
Lets put these in `spaceship/spec/connect_api/spec_helper` (or similar) and have them loaded only for the specs in `connect_api`.

Otherwise they add a lot of stubbing to each spec that maybe totally unrelated.